content/panorama/scripts/custom_game/hero_selection.js

```diff
@@ -348,6 +346,7 @@ function SetupTopBar () {
     return;
```

Comment:
Since `Game.GetMapInfo().map_display_name` is already stored in a variable, you should reference `currentMap`.